pkg/apis/numaflow/v1alpha1/udf.go

```diff
@@ -43,6 +43,10 @@ type UDF struct {
 	Builtin *Function `json:"builtin" protobuf:"bytes,2,opt,name=builtin"`
 	// +optional
 	GroupBy *GroupBy `json:"groupBy" protobuf:"bytes,3,opt,name=groupBy"`
+	// MapStream specifies whether to enable streaming in map udf. If it is
+	// not provided, the default value is set to false.
+	// +optional
+	MapStream bool `json:"mapStream" protobuf:"bytes,4,opt,name=mapStream"`
```

Comment:
Thinking of exposing it through `vertex.metadata.annotation[].xxxx`, let's discuss it.